### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Card:
1. Text-only cards.
2. image cards - vertical
3. image cards - horizontal

Leave the badge.css file.

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+.card-horizontal {
+    padding: 0;
+    width: 30rem;
+    flex-direction: row;
+}
+
+.card-margin-bottom {
+    margin: 0 0 0.5rem 0;
+}
```
Comment (Sahil25061999) [inline components/card/card.css:26]: just a suggestion you can just use margin-bottom.

**Comment by rahulk31** [comment]
A recommendation to align the icon and text vertically centered.
image

Overall looks fine, can be merged. 👍

File: components/card/card.css (hunk 1 above)
Comment (yaswanthmyneni) [reply components/card/card.css:26]: actually, I need that format of margin bottom. so I used it.

**Comment by mehulsatardekar** [comment]
ts

@yaswanthmyneni  a suggestion on hover icon you can add **cursor: "pointer"**. so it will be easy to understand that those cards are dismissable. overall looks good to merge.

**Pull request merged**